frontend/lib/framework/Context.cpp

```
@@ -1191,7 +1191,9 @@ bool Context::queryCanUseSavedResultAndPushIfNot(
 void Context::emitHiddenErrorsFor(const querydetail::QueryMapResultBase* result) {
   CHPL_ASSERT(!result->emittedErrors);
   for (auto& error : result->errors) {
-    reportError(this, error.first.get());
+    if (!error.second) {
```

let's add a comment like `// don't report silenced errors`